components/char_collection/src/char_collection.rs

```diff
@@ -0,0 +1,828 @@
+// Copyright 2019 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+use std::{
+    boxed::Box,
+    char,
+    clone::Clone,
+    cmp::Ordering,
+    convert::Into,
+    error::Error,
+    hash::{Hash, Hasher},
+    iter::Iterator,
+    ops::Range,
+    vec::Vec,
+};
```

Comment:
You don't need to import most of that.

Reply:
agreed and changed, I went through each import to see what the compiler would complain about